float64[6] joint_angles
bool is_moving
